PiwikTracker.php

```diff
@@ -1288,6 +1288,25 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param $proxy
+     * @param int $proxyPort
+     */
+    public function setProxy($proxy, $proxyPort = 80){
```

Comment:
In functions do you mind adding the opening bracket of a function `{` in a new line?